src/signals/incident/components/form/MapSelectors/Asset/AssetSelect.test.tsx

```diff
@@ -176,15 +188,19 @@ describe('AssetSelect', () => {
   })
 
   it('should close the selector component', () => {
-    render(withAssetSelectContext(<AssetSelect {...props} />))
+    render(
+      withAppContext(
+        <Provider store={setupStore()}>
+          <AssetSelect {...props} />
+        </Provider>
+      )
+    )
 
     userEvent.click(screen.getByText(/kies locatie/i))
 
     expect(screen.queryByTestId('assetSelectSelector')).toBeInTheDocument()
 
     userEvent.click(screen.getByTestId('mapCloseButton'))
```

Comment:
zouden we hier niet een gevolg van het click event moeten testen? dus dat er een dispatch getriggered wordt of de close method?